### DISCUSSION TITLE
Survey plugins

### DISCUSSION THREAD

**Discussion by beccaelsworth** [jspsych/jsPsych#2806] [Q&A]
Hi,

Just a quick question - I've been using jspsych version 6.1.0 for a while, and now I'm trying to create a questionnaire which consists of multiple different survey plugins on one page (rather than a new page when I switch between plugins e.g. survey-likert, survey-text, survey-muti-choice). Is this possible using version 6? Or do I need to upgrade my experiments to version 7?

Thank you so so much in advance!

**Comment by jodeleeuw** [comment]
Hi @beccaelsworth,

The [survey plugin](https://www.jspsych.org/7.3/plugins/survey/) is version 7 only.

Version 6.3 does have the [survey-html-form plugin](https://www.jspsych.org/6.3/plugins/jspsych-survey-html-form/), and this plugin is probably compatible with version 6.1.0, but I haven't tested that. If you wanted to stick with version 6.1.0, then I would start by trying that plugin and seeing if it works.

**Reply by beccaelsworth** [reply]
Hi @jodeleeuw 

Thank you so much for your reply. I'll try the survey-html-form plugin - thank you !

Best wishes,
Becca

**Reply by beccaelsworth** [reply]
Hi again @jodeleeuw 

This works well for checkbox and text questions. Do you know if it's possible to include likert scales ?
Thanks so much

**Reply by jodeleeuw** [reply]
The compatible way would probably be to use [radio button groups](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) to create the scale. That's how the jsPsych likert plugins work, but with some CSS styling to make them display horizontally by default.

**Reply by beccaelsworth** [reply]
Thank you so much!
